packages/workspace/tests/implementations/workspace.test.ts

```diff
@@ -0,0 +1,361 @@
+import { take } from 'rxjs/operators';
```

Comment:
From what I see I think it's ok but can you confirm that this tests suit is not couple to a specific configuration service provider ? Meaning that if we decide to replace tomorrow ConfigurationServiceHTTP by the remote configuration service tests will stay exactly the same ?

Reply:
I don't see how it can affect these tests provided the fact that the interface is the same.